aea/test_tools/test_skill.py

```diff
@@ -458,7 +458,7 @@ def prepare_skill_dialogue(
         return dialogue
 
     @classmethod
-    def setup(cls, **kwargs: Any) -> None:
```

Comment:
The base class should still define a `setup` method. Check what can be moved in there..